src/goto-programs/goto_convert.cpp

```diff
@@ -344,14 +344,49 @@ void goto_convertt::convert_label(
 
   goto_programt tmp;
 
-  // magic thread creation label?
+  // magic thread creation label, see ansi-c/library/pthread_lib.c
```

Comment:
It would be good if this comment is now understood, to elaborate here on what it means

Reply:
... and the change should go in a different commit.